inc/plugins/mybbfancybox.php

```diff
@@ -347,7 +347,7 @@ function mybbfancybox_uninstall()
 	update_theme_stylesheet_list(1, false, true);
 
 	// Delete plugin settings in ACP
-	$db->write_query("DELETE FROM ".TABLE_PREFIX."settings WHERE name IN ('mybbfancybox_open_image_urls','mybbfancybox_allowed_extensions','mybbfancybox_include_images_from_urls_into_gallery','mybbfancybox_protect_images','mybbfancybox_watermark','mybbfancybox_watermark_low_resolution_images','mybbfancybox_watermark_resolutions','mybbfancybox_loop','mybbfancybox_infobar','mybbfancybox_arrows','mybbfancybox_thumbs','mybbfancybox_minimize','mybbfancybox_buttons')");
+	$db->write_query("DELETE FROM ".TABLE_PREFIX."settings WHERE name LIKE 'mybbfancybox_%'");
```

Comment:
Doesn't matter though; but would like to drop a comment here that the underscore (`_`) of the LIKE statement `'mybbfancybox_%'` isn't representing literal underscore here, it represents any character 1 length. Hence the meaning here is "Starts with **mybbfancybox** and atleast 13 characters long (12 for the word `fancybox` and 1 for one underscore).  To make it literal underscore you need to escape it.
`'mybbfancybox\_%'` . May be this also work `'mybbfancybox[_]%'`.

Reply:
Good catch. Although, as you suggest, it doesn't matter, I think this should be fixed, simply as a matter of good coding practice.

Reply:
> as a matter of good coding practice

Thats the best part of you @lairdshaw

Reply:
Thanks, @effone. I try.